sys/mips/conf/CHERI_PURECAP_DE4_MFS_ROOT_BASE

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
These should all be very boring files that inherit from a base CHERI config. The BERI configs are already a mess, we don't need to add another whole set of configs that duplicate all the options.